docs/en/observability/observability-introduction.asciidoc

```diff
@@ -1,9 +1,36 @@
 [[observability-introduction]]
 [role="xpack"]
-
-= Observability 
-++++
-<titleabbrev>What is Elastic Observability?</titleabbrev>
-++++
+== What is Elastic Observability? 
```

Comment:
I know this is all placeholder stuff, but we should start thinking about how we want to structure the content.

1 - We definitely want a one to one mapping between topics and HTML output. So one asciidoc file = one HTML file.

2 - To control TOC nesting, we should explore using level offsets. I've done this in the ingest management docs with mixed success. (The strategy worked for +1 and +2 offsets, but broke for anything nested more deeply.) We should follow up with other teams to see if they've had more success, but I think it's the best way forward to make it easier to decouple TOC/navigation from heading levels.